src/core/utils/ifSpecBuilder.h

```diff
@@ -171,7 +171,9 @@ class CStringIdListSpecBuilder {
      * @return required dynamic data size
      */
     std::size_t calcStorageSize() const;
-    std::tuple<const CStringDictionary::TStringId*, TForteUInt8> build(CMixedStorage &paStorage) const;
+
+    template<typename TSize>
```

Comment:
same comment as above, I would go with `Type `or `IndexType`

Reply:
I'll vote for IndexType as written above.